### DISCUSSION TITLE
Current UniProt GO Annotation GAF File is outdated

### DISCUSSION THREAD

**Discussion by davidbio** [geneontology/helpdesk#346] [Q&A] [answered]
Hello GeneOntology-Team,

I haven't had a look at all files, but the latest UniProt all GAF is from June 2021.
http://release.geneontology.org/2021-09-01/annotations/goa_uniprot_all.gaf.gz
Meanwhile, other files are from September 2021, which seems right.
http://release.geneontology.org/2021-09-01/annotations/goa_cow.gaf.gz

I check the release date with the following BASH command.
curl http://release.geneontology.org/2021-09-01/annotations/goa_cow.gaf.gz | zcat | head

Is there no new version available for goa_uniprot_all.gaf.gz?
If this is intended, I think it is confusing to place links to older files in a folder that is named 2021-09-01.

Thank you for looking into this.

**Comment by suzialeksander** [comment, marked as answer]
Hi @davidbio, 

It looks like the file at EBI is from 2021-06, so that line in the header is the date of the source file ftp://ftp.ebi.ac.uk/pub/databases/GO/goa/UNIPROT/goa_uniprot_all.gaf.gz

!
!This file contains all GO annotations and gene product information for proteins in the UniProt KnowledgeBase (UniProtKB),
!ComplexPortal protein complexes, and RNAcentral identifiers.
!
!date-generated: 2021-06-16 17:05 
!generated-by: UniProt
!go-version: http://purl.obolibrary.org/obo/go/releases/2021-06-06/extensions/go-plus.owl

We do have some ongoing discussion about clarifying headers, and although we are not currently planning to change the GAF format we will take this into account when upgrading the GAF 2.2 -> ? .

@alexsign is 2021-06-16 the correct date for the latest goa_uniprot_all.gaf?
